modules/core/src/main/scala/org/scalasteward/core/sbt/SbtAlg.scala

```diff
@@ -93,7 +95,11 @@ object SbtAlg {
         fileAlg.home.map(_ / ".sbt")
 
       def exec(command: Nel[String], repoDir: File): F[List[String]] =
-        ignoreOptsFiles(repoDir)(processAlg.execSandboxed(command, repoDir))
+        if (config.execSandbox) {
+          ignoreOptsFiles(repoDir)(processAlg.execSandboxed(command, repoDir))
+        } else {
+          ignoreOptsFiles(repoDir)(processAlg.exec(command, repoDir))
+        }
 
```

Comment:
Can we move this conditional to `ProcessAlg`? If `config.execSandbox` is false `ProcessAlg.exexSandboxed` should just be an alias for `ProcessAlg.exec`.

Reply:
Made the changes. 👍